config/test.exs

```diff
@@ -1,7 +1,11 @@
 import Config
 
 config :dotcom, :cache, Dotcom.Cache.TestCache
-config :dotcom, :redis, Redis.Mock
-config :dotcom, :redix, Redix.Mock
-config :dotcom, :redix_pub_sub, Redix.PubSub.Mock
+config :dotcom, :redis, Dotcom.Redis.Mock
+config :dotcom, :redix, Dotcom.Redix.Mock
+config :dotcom, :redix_pub_sub, Dotcom.Redix.PubSub.Mock
 config :dotcom, :trip_plan_feedback_cache, Dotcom.Cache.TestCache
+
+config :dotcom, :httpoison, HTTPoison.Mock
+
+config :dotcom, :mbta_api, MBTA.Api.Mock
```

Comment:
This is `MBTA.Api.Mock` instead of `Dotcom.MBTA...` because of the directory structure. We should move all non-web code under `dotcom` at some point. But, for now it's better to be consistent.